verified-reviews/loaders/storeReview.ts

```diff
@@ -0,0 +1,54 @@
+// deno-lint-ignore-file
+import { AppContext } from "../mod.ts";
+import { Review } from "../../commerce/types.ts";
+import { createClient } from "../utils/client.ts";
+
+export type Props = {
+  /**
+   * @title Number of reviews 
+   */
+  limit: number;
+};
+
+/**
+ * @title Opiniões verificadas - Full Review for Store (Ratings and Reviews)
+ */
+export default async function storeReview(
+  _config: Props,
+  _req: Request,
+  ctx: AppContext,
+): Promise<Review[] | null> {
+  const client = createClient({ ...ctx });
+
+  if (!client) {
+    return null;
+  }
+
+  const reviews = await client.storeReview();
+
+  if (!reviews) {
+    return null;
+  }
+
+  let formatReviews: Review[];
+  
+  formatReviews = reviews.map((item) => ({
+    "@type": "Review",
+    author: [
+      {
+        "@type": "Author",
+        name: `${item.firstname} ${item.lastname}`,
+      },
+    ],
+    datePublished: item.review_date,
+    reviewBody: item.review,
+    reviewRating: {
+      "@type": "AggregateRating",
+      ratingValue: Number(item.rate),
+    },
+  }))
+
+  const limitedReviews = formatReviews.slice(0, _config.limit);
```

Comment:
You should use the pagination object instead https://github.com/deco-cx/apps/blob/6a14c13a2ffcd9696d382bb112139e5ccb6d0dfb/verified-reviews/utils/client.ts#L9, is it possible?

Using limit and slice its not so reliable since pagination has limits and you'll reach the limit faster than you think, so always use API pagination instead.

Reply:
In general lines, @pandero7 can we do a fetch already sending the limit?

Reply:
@guitavano @mcandeia In this end-point we don't have pagination. Any sugestion to put smth? Only remove slice?